setup.py

```diff
@@ -1,66 +1,76 @@
 import re
+
 from setuptools import find_packages, setup
 
 install_requires = [
-    'wagtail>=2.0',
-    'user-agents>=1.1.0',
-    'wagtailfontawesome>=1.1.3',
-    'pycountry',
+    "wagtail>=4.1",
+    "user-agents>=1.1.0",
+    "wagtail-font-awesome-svg>=0.0.3",
```

Comment:
When testing this locally, I got the error `ModuleNotFoundError: No module named 'wagtailfontawesome'` when running `./manage.py migrate`. The old migration files still refer to wagtailfontawesome.

Reply:
I'm not sure that now `wagtail-font-awesome-svg>=0.0.3` should have been changed. I think I assumed this package replaced the previous one. But they are different packages.

Did you see the error when running the sandbox or by adding it to a new site?

Reply:
Ah that works now. Thanks. I was previously testing on a wagtail kit build. I see that you have fixed the sandbox and it works fine for me now too. 

I have got some warnings about `DEFAULT_AUTO_FIELD` on my console (for the kit site) though. E.g.

```
 wagtail_personalisation.DayRule: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.DeviceRule: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.OriginCountryRule: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.PersonalisablePageMetadata: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.QueryRule: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.ReferralRule: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.Segment: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.TimeRule: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.UserIsLoggedInRule: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
 wagtail_personalisation.VisitCountRule: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
```

I guess you need to add default_auto_field to your app config?